app/assets/stylesheets/application.css.scss

```diff
@@ -53,8 +54,8 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 
 }
 
-#side-nav .well {
-  padding: 15px;
+#new_transfer{
```

Comment:
Avoid using id selectors<br>Selector `new_transfer` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space